File: OdonSysBackEnd/Resource/Contract/Contract.Admin/Clients/CreateClientRequest.cs
using Utilities.Enums;

namespace Contract.Admin.Clients
{
    public class CreateClientRequest
    {
        public string Name { get; set; }
        public string MiddleName { get; set; }
        public string Surname { get; set; }
        public string SecondSurname { get; set; }
        public string Document { get; set; }
        public string Ruc { get; set; }
        public Country Country { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public string UserId { get; set; }
    }
}

File: OdonSys-SPA/src/app/core/models/api/clients/create-client-request.ts
import { Country } from "../../../../core/enums/country.enum";

export class CreateClientRequest {
  constructor(
    protected name: string,
    protected middleName: string,
    protected surname: string,
    protected secondSurname: string,
    protected phone: string,
    protected country: Country,
    protected email: string,
    protected document: string,
    protected ruc: string
  ) { }
}
